solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -896,6 +900,9 @@ public void load() {
       containerHandlers.getApiBag().registerObject(containerPluginsApi.readAPI);
       containerHandlers.getApiBag().registerObject(containerPluginsApi.editAPI);
 
+      // get the placement plugin
```

Comment:
I'd rather comment "get the placement plugin **factory**"
And possibly specify what's the plugin factory lifecycle wrt configuration? Now that everything is a bit more implicit than it previously was, I don't get (yet...) when exactly the configuration is passed. I believe comments related to this would be useful.